src/panels/lovelace/components/hui-image.ts

```diff
@@ -159,12 +161,25 @@ export class HuiImage extends LitElement {
   }
 
   protected updated(changedProps: PropertyValues): void {
+    this._checkConnection();
+
     if (changedProps.has("cameraImage") && this.cameraView !== "live") {
       this._updateCameraImageSrc();
       this._startUpdateCameraInterval();
     }
   }
 
+  private async _checkConnection(): Promise<void> {
+    if (this.hass!.connected) {
+      await this._updateCameraImageSrc();
+      this._startUpdateCameraInterval();
```

Comment:
We are going to restart the time now on every update? Shouldn't we just do this when the connection changes from not connected to connected?

Reply:
Also we are doing this again on line 167 after doing it here